drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -171,11 +176,16 @@ local function do_configure(driver, device)
   if #auto_eps ~= 0 and device:get_field(setpoint_limit_device_field.MIN_DEADBAND) == nil then
     setpoint_limit_read:merge(clusters.Thermostat.attributes.MinSetpointDeadBand:read())
   end
+  if #temp_eps ~= 0 then
```

Comment:
Similarly to what you've done in the switch and sensor drivers, let's move any limit reads to the `init` function so that we will update devices already in the field as well! `doConfigure` in this instance will essentially only run for new device adds, but moving this logic to `init` will allow us to set these limits for devices already in the field without requiring a fresh onboard.

You can move the TemperatureMeasurement limit reads that you added, as well as the Thermostat setpoint reads that are also in this `doConfigure` function. This way, all of these capability ranges can be set for devices already joined to the platform.

Reply:
Moved all of the limit reads form `doConfigure` into `init` in c33ee3ad996ac550630fc72a2fd778f617096866. Updated the test cases as needed.